README.md

```diff
@@ -1,252 +1,173 @@
-```
-
-Options:
+import {parseNumber} from '@brightspace-ui/intl/lib/number.js';
 
-- **locale**: see [overriding locale data](#Overriding Locale Data)
-
-**Example:**
-
-```javascript
-var parser = new d2lIntl.NumberParse('fr-CA');
-console.log(parser.parse('-8 942,39')); // -> -8942.39
+const value = parseNumber('-8 942,39'); // -> -8942.39 in fr-CA
 ```
 
 ## Date/Time Formatting
 
-Dates and times can be formatted in the user's locale using the `DateTimeFormat` class. It behaves similar to the ECMA-402 [Intl.DateTimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat) class.
+Dates and times can be formatted in the user's locale using `formatDate`, `formatTime` and `formatDateTime`.
 
-Syntax:
+Combined dates and times are formatted using `formatDateTime`:
 
 ```javascript
-var formatter = new d2lIntl.DateTimeFormat(locales[, options]);
+import {formatDateTime} from '@brightspace-ui/intl/lib/dateTime.js';
+
+const date = formatDateTime(
+	new Date(2015, 8, 23, 14, 5),
+	[options]
+); // -> '2015-09-23 14:05' in sv-SE
 ```
 
 Options:
+- **format**: pattern to use when rendering the date-time; default is `'short'`.
+  - **full**: long weekday, month names and timezone. e.g. `'Wednesday, September 23, 2015 1:25 PM EST'`
+  - **medium**: long month names. e.g. `'September 23, 2015 1:25 PM'`
+  - **short**: abbreviated date format. e.g. `'9/23/2015 1:25 PM'`
 
-- **locale**: see [overriding locale data](#Overriding Locale Data)
-- **format**: which pattern to use when rendering the date-time; default is "short".
-  - **full**: long weekday, month names and timezone. e.g. "Wednesday, September 23, 2015 1:25 PM EST"
-  - **medium**: long month names. e.g. "September 23, 2015 1:25 PM"
-  - **short**: abbreviated date format. e.g. "9/23/2015 1:25 PM"
-  - **monthYear**: month and year only. e.g. "September 2015"
-  - **monthDay**: month and day only. e.g. "September 23"
-  - **longDayOfWeek**: long weekday only. e.g. "Wednesday"
-  - **shortDayOfWeek**: short weekday only. e.g. "Wed"
-  - **longMonth**: long month only. e.g. "September"
-  - **shortMonth**: short month only. e.g. "Sep"
-
-All the date and time formatting methods take a [JavaScript Date object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date) as input.
-
-To format a **date and time**, use the `format` method:
+To format a **date only** (without the time portion), use `formatDate`:
 
 ```javascript
-var formatter = new d2lIntl.DateTimeFormat('sv-SE');
-var time = formatter.format(
-	new Date(2015, 8, 23, 14, 5)
-); // -> 2015-09-23 14:05
-```
+import {formatDate} from '@brightspace-ui/intl/lib/dateTime.js';
 
-To format a **time only** (without the date portion), use the `formatTime` method:
-
-```javascript
-var formatter = new d2lIntl.DateTimeFormat('ko');
-var time = formatter.formatTime(
-	new Date(2015, 8, 23, 14, 5)
-); // -> 오후 14:05
+const value = formatDate(
+	ew Date(2015, 8, 23),
```

Comment:
nit: `ew`

Reply:
and missing `'` on the line below around `full`

Reply:
lol thank-you